test/unit/com/unifina/controller/api/CanvasApiControllerSpec.groovy

```diff
@@ -40,6 +41,7 @@ class CanvasApiControllerSpec extends Specification {
 	static doWithSpring = {
 		springSecurityService(SpringSecurityService)
 		userService(UserService)
+		sessionService(SessionService)
```

Comment:
Rather use a mock than the real service? (Yeah, this old test uses real versions of `SpringSecurityService` and `UserService` while it shouldn't - no need to fix that retroactively, but better not keep expanding the set of collaborators)